src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -466,3 +466,50 @@ def upload_string(to_upload, media_type=None, keep_open=False, wait_on_close=Fal
         handler.close(block=wait_on_close, **remaining_kwargs)
 
     return handler
+
+def download_folder(project, destdir, folder="/", chunksize=dxfile.DEFAULT_BUFFER_SIZE):
```

Comment:
Thanks, that's a good point. Is it easy to move the common stuff into an internal method (that supports the cache) and have download_folder be a thin wrapper that disables or prevents access to the use of the caching features? That would allow us to share most of the code but defer this question about whether users of this API need caching.

If it's too complicated, don't bother. Thanks!